src/ticketmap/index.tsx

```diff
@@ -97,6 +100,48 @@ const $availableTicketGroups = createDeepEqualSelector(
     }).filter(ticketGroup => ticketGroup)
 )
 
+const $priceSortedTicketGroups = createDeepEqualSelector(
+  $ticketGroups,
+  ticketGroups => ticketGroups.sort((a, b) => a.retail_price - b.retail_price)
+)
+
+const $sectionPercentiles = (state: State, props: Props) => props.sectionPercentiles
```

Comment:
Why does this have state _and_ props params?

Reply:
This function signature is required for selectors that access props. It allows other selectors to compose this one without fiddling with parameter order. For example:

```js
import { createSelector } from 'reselect'

const $sectionPercentiles = (state: State, props: Props) => props.sectionPercentiles

const $selectedPercentile = createSelector(
  $sectionPercentiles,
  (state: State) => state.selectedPercentileId,
  (percentiles, id) => percentiles[id]
)

/* ... */

render() {
  return <div>{$selectedPercentile(state, props)}</div>
}
```

Reply:
Ah I see, the selector params are passed into each of the embedded functions (except for the last one) in the same order as they were passed into the selector. The first function needs props, and the second function needs state, so the second function can have a signature without the second param but the first function can't have a signature without the first.

Reply:
Spot on!